views/default/page/elements/comments.php

```diff
@@ -19,36 +19,29 @@
 	$limit = elgg_trigger_plugin_hook('config', 'comments_per_page', [], 25);
 }
 
-$id = '';
-if (isset($vars['id'])) {
-	$id = "id=\"{$vars['id']}\"";
-}
-
-$class = 'elgg-comments';
-if (isset($vars['class'])) {
-	$class = "$class {$vars['class']}";
-}
+$attr = [
+	'id' => elgg_extract('id', $vars, 'comments'),
+	'class' => (array) elgg_extract('class', $vars, []),
+];
+$attr['class'][] = 'elgg-comments';
```

Comment:
Ok. Just something that came to mind. Wouldn't block it if I could :)
On Apr 20, 2015 5:34 PM, "Steve Clay" <notifications@github.com> wrote:

> In views/default/page/elements/comments.php
> <https://github.com/Elgg/Elgg/pull/8226#discussion_r28699274>:
>
> > @@ -19,36 +19,29 @@
> >  	$limit = elgg_trigger_plugin_hook('config', 'comments_per_page', [], 25);
> >  }
> >
> > -$id = '';
> > -if (isset($vars['id'])) {
> > -	$id = "id=\"{$vars['id']}\"";
> > -}
> > -
> > -$class = 'elgg-comments';
> > -if (isset($vars['class'])) {
> > -	$class = "$class {$vars['class']}";
> > -}
> > +$attr = [
> > +	'id' => elgg_extract('id', $vars, 'comments'),
> > +	'class' => (array) elgg_extract('class', $vars, []),
> > +];
> > +$attr['class'][] = 'elgg-comments';
>
> FYI lots of class reodering went into 1.11: afcf8ff
> <https://github.com/Elgg/Elgg/commit/afcf8fff0b764ea79ef48efa5e97e86c669bc018>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8226/files#r28699274>.
>